src/app/community-page/sub-collection-list/themed-community-page-sub-collection-list.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ThemedComponent } from '../../shared/theme-support/themed.component';
+import { CommunityPageSubCollectionListComponent } from './community-page-sub-collection-list.component';
+import { Component, Input } from '@angular/core';
+import { Community } from '../../core/shared/community.model';
+
+@Component({
+  selector: 'ds-themed-community-page-sub-collection-list',
+  styleUrls: [],
+  templateUrl: '../../shared/theme-support/themed.component.html',
+})
+export class ThemedCollectionPageSubCollectionListComponent extends ThemedComponent<CommunityPageSubCollectionListComponent> {
+  @Input() community: Community;
+  @Input() pageSize: number;
+  protected inAndOutputNames: (keyof CommunityPageSubCollectionListComponent & keyof this)[] = ['community', 'pageSize'];
+
+  protected getComponentName(): string {
+    return 'CommunityPageSubCollectionListComponent';
+  }
+
+  protected importThemedComponent(themeName: string): Promise<any> {
+    return import(`../../../themes/${themeName}/app/community-page/sub-community-list/community-page-sub-collection-list.component`);
```

Comment:
Looks like this path is wrong

Reply:
Sorry this took so long, @artlowel . The path is fixed.

I was going to say that this PR  still needs to wait on #1750 ... but it looks like that was just merged. I'll update.